<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>641</width>
    <height>343</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pwrButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>ВЫКЛ.</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>0</y>
     <width>511</width>
     <height>341</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="mechaduinoContainer"/>
  </widget>
  <widget class="QPushButton" name="tuneButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Tune</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="freqEdit">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>120</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>14.225</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
